prow/cmd/deck/runlocal

```diff
@@ -13,25 +13,72 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -o errexit
+set -o nounset
+set -o pipefail
+
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd "${DIR}"
-cd localdata
-HOST=${1:-"https://prow.k8s.io"}
-if [[ "${HOST}" == "openshift" ]]; then
-	HOST="https://prow.ci.openshift.org"
+
+fetch() {
+  echo "* $@" >&2
+  curl -s "$@" 
+}
+
+
+update-pregenerated() {
+
+  local readonly HOST=$1
+
+  echo "Updating pregenerated data from ${HOST}: " >&2
+
+  cd localdata
+  fetch "${HOST}/plugin-help.js?var=allHelp" > plugin-help.js
+  fetch "${HOST}/pr-data.js" > pr-data.js
+  fetch "${HOST}/prowjobs.js" > prowjobs.json
+  fetch "${HOST}/tide-history.js?var=tideHistory" > tide-history.js
+  fetch "${HOST}/tide.js?var=tideData" > tide.js
+  cd ..
+  echo "DONE"
+}
+
+if [[ "${1:-}" != '--skip-pregenerated' ]]; then
```

Comment:
would be better to allow out-of-order args. understand that this script doesn't have to be prod grade, so up to you to decide

Reply:
Ack